gateway-service/src/test/java/org/zowe/apiml/gateway/security/login/saf/MockPlatformUserTest.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ */
+
+package org.zowe.apiml.gateway.security.login.saf;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.zowe.apiml.security.common.auth.saf.PlatformReturned;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class MockPlatformUserTest {
```

Comment:
Don't you want to use Nested here? It would be quite handy and shorten the names here.

Reply:
added nested classes